export const editUser = async(values)=>{
    try{

    }catch(error){
        console.error(error);
        res.status(500).json({ error: "Internal server error" });
    }
}


export const addImage = async(values)=>{
    try{

    }catch(error){
        console.error(error);
        res.status(500).json({ error: "Internal server error" });
    }
}


export const changePassword = async(values)=>{
    try{

    }catch(error){
        console.error(error);
        res.status(500).json({ error: "Internal server error" });
    }
}


export const removeUser = async(values)=>{
    try{

    }catch(error){
        console.error(error);
        res.status(500).json({ error: "Internal server error" });
    }
}
